submissions/tosinKolawole/html-forms/product-page.html

```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
```

Comment:
You don't have t create a `product-page.html` file inside the `html-forms` folder. Kindly link the `product-page.html` that is in the `html` folder

Reply:
I don't really understand what I'm supposed to do. Can you kindly expatiate

Reply:
During your `html` task, you created a `product page` file and you placed that file in your `html` folder I guess. For this task, you still went ahead to create the same `product-page.html` file in your `html-forms` folder. This wrong. You don't have to create another one in your `html-forms` folder. Your product page should be located in your `html` folder not anywhere else.